src/libcore/cell.rs

```diff
@@ -10,6 +10,23 @@
 
 //! Shareable mutable containers.
 //!
+//! Rust memory safety is based on this rule: Given an object `T`, it is only possible to
+//! have one of the following:
+//!
+//! - Having several immutable references (`&T`) to the object (also know as Aliasing).
```

Comment:
`s/know as/known as/`

`s/Aliasing/**aliasing**/` . Capitalization of common nouns is unnatural in English, and when it is done, it generally implies that the capitalized noun has a much more specific definition than the lowercase noun (e.g. "Undefined Behavior" is not the same thing as "undefined behavior").  This is unnecessary for a word like "aliasing" which is already uncommon.